src/app/public/modules/colorpicker/colorpicker.component.ts

```diff
@@ -76,22 +76,50 @@ import {
 
 let componentIdIndex = 0;
 
+/**
+ * Provides a SKY UX-themed replacement for the HTML `input` element with `type="color"`.
+ * The value that users select is driven through the `ngModel` attribute specified on
+ * the `input` element.
+ */
 @Component({
   selector: 'sky-colorpicker',
   templateUrl: './colorpicker.component.html',
   styleUrls: ['./colorpicker.component.scss']
 })
 
 export class SkyColorpickerComponent implements OnInit, OnDestroy {
+  /**
+   * Fires when users select a color in the colorpicker. Both events
+   * return an object with the following properties:
+   * ```{
+   * hslaText: string
+   * rgbaText: string
+   * cmykText: string
+   * hsva: { hue:  number, saturation: number, value: number,      alpha: number}
+   * rgba: { red:  number, green: number,      blue: number,       alpha: number}
+   * hsla: { hue:  number, saturation: number, lightness: number,  alpha: number}
+   * cmyk: { cyan: number, magenta: number,    yellow: number,     key: number  }
+   * hex:  string
+   * }```
+   */
```

Comment:
This section is rendering as one long line instead of starting new lines. I looked it up and tested it, but to get it to render correctly, it'll need to look like the example below. I wasn't able to figure out how to get the indent, but I figured getting the new lines working is better than nothing.
```
   * Fires when users select a color in the colorpicker. Both events
   * return an object with the following properties:
   * <pre>
   * {
   * hslaText: string 
   * rgbaText: string
   * cmykText: string
   * hsva: { hue: number, saturation: number, value: number, alpha: number}
   * rgba: { red: number, green: number, blue: number, alpha: number}
   * hsla: { hue: number, saturation: number, lightness: number, alpha: number}
   * cmyk: { cyan: number, magenta: number, yellow: number, key: number}
   * hex: string
   * } <pre>
   */
```

Reply:
Let's kill it.